d3a_api_client/redis_aggregator.py

```diff
@@ -44,13 +44,13 @@ def _connect_to_simulation(self, is_blocking=True):
 
     def _subscribe_to_response_channels(self):
         event_channel = f'external-aggregator/*/{self.aggregator_uuid}/events/all'
-        channel_dict = {"aggregator_response": self._aggregator_response_callback,
-                        event_channel: self._events_callback_dict,
+        channel_dict = {event_channel: self._events_callback_dict,
                         f"external-aggregator/*/{self.aggregator_uuid}/response/batch_commands":
                             self._batch_response,
                         }
 
         self.pubsub.psubscribe(**channel_dict)
+        self.pubsub.subscribe(**{"aggregator_response": self._aggregator_response_callback})
```

Comment:
Any reason for that? ´psubscribe´ can also handle non-patterns.

Reply:
I'm not sure why I wasn'T getting response callback while being part of `psubscribe` while debugging issue reported [here](https://gridsingularity.atlassian.net/browse/D3ASIM-3203?focusedCommentId=21970)

Reply:
Hmm, are you sure that these are connected?

Reply:
I just tested it again, with pattern subscription wasn't able to get response whereas by exact subscription was getting response accordingly.

Reply:
I just had a look in the docs and there is no place where they explain that `psubscribe` can only subscribe to patterns. I remember testing this in the past and it worked. We also do in other places.
Please have another look, there might be something else wrong here.